java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -191,30 +199,40 @@ private static boolean isMethodInvocationNode(ExplodedGraph.Node node) {
     return false;
   }
 
-  private static int correspondingArgumentIndex(SymbolicValue candidate, ExplodedGraph.Node invocationNode) {
+  private static int correspondingArgumentIndex(Set<SymbolicValue> candidates, ExplodedGraph.Node invocationNode) {
     MethodInvocationTree mit = (MethodInvocationTree) invocationNode.programPoint.syntaxTree();
     List<SymbolicValue> arguments = argumentsUsedForMethodInvocation(invocationNode, mit);
-    return arguments.indexOf(candidate);
+    // FIXME SONARJAVA-2076 whatif multiple candidates match some of the arguments?
```

Comment:
This one ought to be fixed and is (if i'm correct) not related to SONARJAVA-2076. See my comment on the call site of correspondingArgumentIndex.